docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/identifying-policy-in-use.adoc

```diff
@@ -0,0 +1,27 @@
+= Identifying Policy in-use
+
+In {ProductName}, there are two places where Enterprise Contract Policies are used by default,
+integration tests and release pipelines.
```

Comment:
This feels like the place to add the content I brought up above. We state that there are two places that policies are used by default. We describe each in isolation, so we should also mention how they interact jointly.

Somewhere on this page, you can potentially illustrate that the _same_ policy can be used for an ITS and a RPA assuming that hte policy is present on-cluster somewhere.

Reply:
Seems like a good idea. I don't think it's within the scope of this PR (which is associated with a feature). Can you file an issue for this to be done later on?